apps/consulting/posts/categories.json

```
@@ -0,0 +1,46 @@
+[
+  {
+    "name": "Data Visualization",
+    "value": "data-visualization"
+  },
+  {
+    "name": "Open Source Software",
+    "value": "open-source-software"
+  },
+  {
+    "name": "Training",
+    "value": "training"
+  },
+  {
+    "name": "PyData Ecosystem",
+    "value": "pydata-ecosystem"
+  },
+  {
+    "name": "Artificial Intelligence",
+    "value": "artificial-intelligence"
+  },
+  {
+    "name": "Optimization",
+    "value": "optimization"
+  },
+  {
+    "name": "Jupyter",
+    "value": "jupyter"
+  },
+  {
+    "name": "Scalable Computing",
+    "value": "scalable-computing"
+  },
+  {
+    "name": "Infrastructure & HPC",
+    "value": "Infrastructure-hpc"
```

slugs should probably all be all lower case, right?

```suggestion
    "value": "infrastructure-hpc"
```